.gitignore

```diff
@@ -3,6 +3,7 @@
 /db
 /deps
 /*.ez
+.DS_Store
```

Comment:
This is ok, but you should also add it to your global `.gitignore` as not all projects will have this.